ui/desktop/src/utils/generateId.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Generate a random ID string
+ * @returns A random string ID
+ */
+export function generateId(): string {
```

Comment:
This looks duplicated from the implementation in `types/message.ts` - perhaps remove this file?